docs/core-concepts/model-basics.md

```diff
@@ -308,18 +308,22 @@ sequelize.define('User', {
 });
 ```
 
-Some special values, such as `DataTypes.NOW`, are also accepted:
+It is possible to use `fn` to use a native SQL function as the default value:
 
 ```js
 sequelize.define('Foo', {
-  bar: {
-    type: DataTypes.DATETIME,
-    defaultValue: DataTypes.NOW
-    // This way, the current date/time will be used to populate this column (at the moment of insertion)
+  myUuid: {
+    type: DataTypes.UUID,
+    defaultValue: fn('uuid_generate_v4'),
```

Comment:
omg this is amazing :o

Reply:
confirmed 
![image](https://user-images.githubusercontent.com/1280915/158879876-9b8d103d-63a4-4d86-8474-210cec0c3e77.png)